readme.md

```diff
@@ -1,7 +1,7 @@
 # Hello world
-1. Replace `<your_account>` with your Github username in the links
-    - [DEMO LINK](https://<your_account>.github.io/layout_hello-world/) <br>
-    - [TEST REPORT LINK](https://<your_account>.github.io/layout_hello-world/report/html_report/)
+1. Replace `<starzhitskiy>` with your Github username in the links
+    - [DEMO LINK](https://<starzhitskiy>.github.io/layout_hello-world/) <br>
+    - [TEST REPORT LINK](https://<starzhitskiy>.github.io/layout_hello-world/report/html_report/)
```

Comment:
```suggestion
    - [DEMO LINK](https://starzhitskiy.github.io/layout_hello-world/) <br>
    - [TEST REPORT LINK](https://starzhitskiy.github.io/layout_hello-world/report/html_report/)
```
You don't need to put these brackets to the URL. You just need to insert your name to make a correct link to your page